### ISSUE TITLE
pip install regrid issue

### ISSUE THREAD

**Issue by matt-long** [NCAR/esmlab#117]
Building environments using a conda environment file with the following pip directive

```
git+https://github.com/NCAR/esmlab.git
```
leads to the following error
```
Pip subprocess error:
  Could not find a version that satisfies the requirement esmlab-regrid (from esmlab==2019.3.16+140.gd3ce9a5->-r /gpfs/u/home/mclong/codes/NCAR-pangeo-tutorial/environments/condaenv.grstg34t.requirements.txt (line 4)) (from versions: )
No matching distribution found for esmlab-regrid (from esmlab==2019.3.16+140.gd3ce9a5->-r /gpfs/u/home/mclong/codes/NCAR-pangeo-tutorial/environments/condaenv.grstg34t.requirements.txt (line 4))
```
cc @andersy005

**Comment by matt-long** [comment]
Adding 
```
git+https://github.com/NCAR/esmlab-regrid.git
```
seems to solve the problem.

**Comment by andersy005** [comment]
@matt-long, thank you for reporting this. I ran into this issue before, and I meant to fix this issue a few days ago but I ended up forgetting. 

It is caused by the presence of esmlab-regrid in https://github.com/NCAR/esmlab/blob/master/requirements.txt#L4 and because esmlab-regrid is not on PyPI yet, pip gives up and throws the error. 

For the time being, the solution you propose should suffice until we have esmlab-regrid on PyPI.
